<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_4</class>
 <widget class="QWidget" name="page_4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pBack_page4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/arrows_back_icon.png</normaloff>:/new/prefix1/arrows_back_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>90</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aa삐삐롱</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 40px;
background-color: rgba(89, 141, 204);
color:white;</string>
   </property>
   <property name="text">
    <string>DataBase</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>761</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDateTimeEdit" name="pdateTimeEditFrom">
        <property name="date">
         <date>
          <year>2022</year>
          <month>9</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="pdateTimeEditTo">
        <property name="date">
         <date>
          <year>2022</year>
          <month>9</month>
          <day>30</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="pTableWidget">
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>252</number>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date/Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>On/Off</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pPBSearch">
      <property name="text">
       <string>search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pPBDelete">
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
